docs/book/src/component-config-tutorial/updating-main.md

```diff
@@ -0,0 +1,22 @@
+# Updating main
+
+Once you have defined your new custom component config type, we need to change
+the options parsing in `main.go` to use this new type. To do this we'll chain
+`OfKind` onto `ctrl.ConfigFile()` and pass in a pointer to the config kind.
+
+```go
+var err error
```

Comment:
It would be helpful to have imports shown in this code example, or link to the source.

Reply:
Good call, I also need to make sure schemes have been added.